docs/source/index.md

```diff
@@ -0,0 +1,53 @@
+# Neuroinformatics Unit Python cookiecutter
+
+The Neuroinformatics Unit (NIU) is a **Research Software Engineering** team based at the **Sainsbury Wellcome Centre** and the **Gatsby Computational Neuroscience Unit**.
+
+The NIU is dedicated to the development of high quality, accurate, robust, easy to use and maintainable open-source software for neuroscience and machine learning. We collaborate with researchers and other software engineers to advance research in the two research centres and make new algorithms and tools available to the global community,[Read more](https://neuroinformatics.dev/index.html).
```

Comment:
I don't think we need to describe the team here, this site should be about the tool, and just link to our main site.